server/src/rest.rs

```diff
@@ -0,0 +1,109 @@
+//! Public API for the svc-cargo server
+//! REST Endpoints for client applications
+
+// use std::{
+//     sync::Arc,
+//     // collections::HashMap
+// };
+
+use axum::{response::IntoResponse, Json};
+use hyper::{HeaderMap, StatusCode};
+mod types {
+    include!("../../openapi/types.rs");
+}
+pub use types::*;
+
+/// Get All Vertiports in a Region
+///
+/// List all Vertiport items from svc-storage
+#[utoipa::path(
+    get,
+    path = "/cargo/region",
+    request_body = RegionQuery,
+    responses(
+        (status = 200, description = "List all cargo-accessible vertiports successfully", body = [Vertiport])
+    )
+)]
+pub async fn query_vertiports(Json(payload): Json<RegionQuery>) -> Json<Vec<Vertiport>> {
+    // let vertiports = store.lock().await.clone();
+    // TODO Query svc-storage
+    println!("{:?}", payload);
+
+    Json(vec![])
+}
+
+/// Search FlightOptions by query params.
+///
+/// Search `FlightOption`s by query params and return matching `FlightOption`s.
+#[utoipa::path(
+    get,
+    path = "/cargo/query",
+    request_body = FlightQuery,
+    responses(
+        (status = 200, description = "List possible berths", body = [FlightOption])
```

Comment:
What does berths mean in this context? I thought that's more for ships? We should make things less ambiguous as per writing style guide.

Reply:
It is indeed confusing, will update

Reply:
Changes to "flights": https://github.com/Arrow-air/svc-cargo/pull/1/commits/9cdbe864c593acdc4bf89bcc76db1d2aa2e97227